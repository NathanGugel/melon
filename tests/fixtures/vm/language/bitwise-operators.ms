io.print(1 << 8);
io.print(1 << 32);
io.print((1 << 32) >> 32);
io.print(((1 << 32) - 1) & 255);
io.print(((1 << 32) - 1) & ((1 << 16) - 1));
io.print(^9);
io.print(^(1 << 8) + 1);
io.print(10 ^^ 10);
io.print(1 ^^ 2);
io.print(255 | (255 << 8));
io.print(1 | 2);