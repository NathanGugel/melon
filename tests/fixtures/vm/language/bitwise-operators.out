256
4294967296
1
255
65535
-10
-256
0
3
65535
3
